x-pack/legacy/plugins/index_management/common/lib/index.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+export {
```

Comment:
On the review step, there is the `JSON Request` tab, which shows JSON in the format as it would be sent to ES.

Reply:
Got it 👍

Reply:
@sebelga In general we will need serialization either client side or accessible to the client for the reason Alison mentioned, because our apps will expose the underlying Elasticsearch requests to the user wherever possible/useful.